src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/IDeclarationInfo.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.FindSymbols
+{
+    internal interface IDeclarationInfo
```

Comment:
Can you document what this type is conceptually? Otherwise it just looks like a lazy way to write list.